src/material-experimental/mdc-slide-toggle/slide-toggle-config.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {InjectionToken} from '@angular/core';
+
+
+/** Default `mat-slide-toggle` options that can be overridden. */
+export interface MatSlideToggleDefaultOptions {
```

Comment:
I wanted to avoid bringing in `material/slide-toggle` and this was the only place that was using it.

Reply:
Ok, I don't think its something we specifically need to avoid, `@angular/material-experimental` has a peer dep on `@angular/material` anyways. But its also not a ton of code in this case so ¯\\_(ツ)_/¯